client/src/main/java/io/pravega/client/ClientConfig.java

```diff
@@ -230,6 +244,20 @@ public ClientConfig build() {
             if (controllerURI == null) {
                 controllerURI = URI.create("tcp://localhost:9090");
             }
+            String scheme = controllerURI.getScheme();
```

Comment:
It doesn't seem like the equals checks below are needed, if this properly returned null in the event of `host:port`.

Reply:
resolved it.